action.yml

```diff
@@ -2,9 +2,7 @@
 
 name: Set up MATLAB
 description: >-
-  Install MATLAB on a Linux-based GitHub-hosted agent. Currently, this action is
-  available only for public projects and does not include transformation
-  products, such as MATLAB Coder and MATLAB Compiler.
+  Setup MATLAB on a Linux-based GitHub-hosted agent
```

Comment:
Install MATLAB on a GitHub-hosted runner

Reply:
I am wary of the term install. Given "Setup" is standard GitHub nomenclature, I am favor of going with that

Reply:
Interesting. In this case, let's use the verb form "Set up".

Reply:
Should this be switched to "Set up" per Houman?

Reply:
Okay so it seems that the standard across most actions is "Setup" (as opposed to "Set up"). To stay consistent with other actions in the same UI, I'll go "Setup" for now 🤔

Reply:
Ok, thanks for checking this 😥

Reply:
Hi, Johan. Using "setup" as a verb is incorrect and weird. If you like to use it, please use it as a noun. In other words, change "Setup MATLAB ..." to "MATLAB setup ...".

My preference, however, is to use a verb phrase (as in the actions listed here: https://github.com/marketplace?type=actions&query=set+up).

Reply:
TL;DR - Why dont we compromise and go with the verb phrase "Set up" or "Sets up". I think that we might be in the middle of a transition where Setup as a verb might be getting more allowable in the language, given the language of tech does seem to use it more often. 

In  addition to Houman's link this one is also interesting:
https://github.com/marketplace?type=actions&query=setup

Some very notable projects use setup as a verb (Node, Go) but others are using Setup in the title but using "Set up" or "Sets up" in the description. Its all over the place, abut even if I am right and we are in a transition, I hitch Houman's is the only proposal that is correct either way.

Interesting tech style guide also supports Houman's preference
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/set-up-setup

Someday we will we learn that Houman is *always* on point.

Reply:
To be clear, I think we should go with the title as Setup (this is standard for actions, and it is a title like Microsoft's Setup.exe case), but the description as Set up or Sets up

Reply:
Sounds good, Andy. This way we will be aligned with the marketplace community :).

Reply:
Awesome discussion. I've gone with "Setup" in the title, and "Set up" in the description 😄